nvidia-nano/01_ansible_nvidia-nano_base.yaml

```diff
@@ -1,8 +1,613 @@
+      ansible.builtin.copy:
+        src: "{{ playbook_dir }}/ROOTFS/etc/dnsmasq.d/10-waggle-base.conf"
+        dest: /etc/dnsmasq.d/10-waggle-base.conf
+
+    - name: Configure dnsmasq to create the device's internal 10.31.81.1/24 network by copying dnsmasq.service.d directory.
+      ansible.builtin.copy:
+        src: "{{ playbook_dir }}/ROOTFS/etc/systemd/system/dnsmasq.service.d"
+        dest: /etc/systemd/system
+
+    - name: Set up k3s service override
+      ansible.builtin.copy:
+        src: "{{ playbook_dir }}/ROOTFS/etc/systemd/system/k3s.service.d"
+        dest: /etc/systemd/system
+
+    - name: Setup the local docker registry mirrors (as the k3s config uses them)
+      ansible.builtin.copy:
+        src: "{{ playbook_dir }}/ROOTFS/etc/systemd/system/waggle-registry-local.service"
+        dest: /etc/systemd/system/waggle-registry-local.service
+
+    - name: Add fallback DNS
+      ansible.builtin.copy:
+        src: "{{ playbook_dir }}/ROOTFS/etc/systemd/resolved.conf.d/10-waggle-resolved.conf"
+        dest: /etc/systemd/resolved.conf.d/10-waggle-resolved.conf
+
+    - name: Add wifi dongle support by copying ROOTFS/etc/udev/rules.d/10-waggle.rules to system
+      ansible.builtin.copy:
+        src: "{{ playbook_dir }}/ROOTFS/etc/udev/rules.d/10-waggle.rules"
+        dest: /etc/udev/rules.d/10-waggle.rules
+
+    - name: Add wifi dongle support by copying ROOTFS/etc/modprobe.d/rtl8821cu.conf to system
+      ansible.builtin.copy:
+        src: "{{ playbook_dir }}/ROOTFS/etc/modprobe.d/rtl8821cu.conf"
+        dest: /etc/modprobe.d/rtl8821cu.conf
+
+    - name: Add wifi dongle support by copying ROOTFS/etc/modprobe.d/rtl8822bu.conf to system
+      ansible.builtin.copy:
+        src: "{{ playbook_dir }}/ROOTFS/etc/modprobe.d/rtl8822bu.conf"
+        dest: /etc/modprobe.d/rtl8822bu.conf
+
+    - name: Add waggle hotspot support by copying ROOTFS/etc/NetworkManager/system-connections/wifi-waggle to system
+      ansible.builtin.copy:
+        src: "{{ playbook_dir }}/ROOTFS/etc/NetworkManager/system-connections/wifi-waggle"
+        dest: /etc/NetworkManager/system-connections/wifi-waggle
+
+    - name: Enable NetworkManager Connectivity checks by copying ROOTFS/etc/NetworkManager/conf.d/99-connectivity.conf to system
+      ansible.builtin.copy:
+        src: "{{ playbook_dir }}/ROOTFS/etc/NetworkManager/conf.d/99-connectivity.conf"
+        dest: /etc/NetworkManager/conf.d/99-connectivity.conf
+
+    - name: Add ssh known hosts to system
+      ansible.builtin.copy:
+        src: "{{ playbook_dir }}/ROOTFS/etc/ssh/ssh_known_hosts"
+        dest: /etc/ssh/ssh_known_hosts
+
+    - name: Reload Daemon
+      ansible.builtin.systemd:
+        daemon_reload: yes
+
+- name: Configure Docker to use External Media
+  hosts: all
+  tasks:
+    #Filesystem gets created when docker's first command is ran
+    - name: Start up docker to create filesystem
```

Comment:
this doesnt actually start docker.  instead we should just stop docker. remove the entire docker folder, as we dont care what existed before and create a new folder in the /media/plugin-data and symlink to that.  let docker start from scratch here.

Reply:
later update